### ISSUE TITLE
pip install torchvision not pulling in the latest code

### ISSUE THREAD

**Issue by vastech73** [pytorch/vision#761]
We noticed an issue with torchvision wheel distribution not reflecting some of the latest changes.

The required (below) code is missing with the wheel distribution when we do “ pip install torchvision”

https://github.com/pytorch/vision/commit/6434deadb3566999abe97bab963fbdc092d9028c#diff-d882c698812f057dfd09020d4a0610a2

vvisvana@vvisvana-linux:~/.local/lib/python2.7/site-packages/torchvision/models$ pip show torchvision
Name: torchvision
Version: 0.2.1
Summary: image and video datasets and models for torch deep learning
Home-page: https://github.com/pytorch/vision
Author: PyTorch Core Team
Author-email: soumith@pytorch.org
License: BSD
Location: /home/vvisvana/.local/lib/python2.7/site-packages
Requires: six, numpy, torch, pillow

Pls see the alexnet.py file that is missing the above change :

import torch.nn as nn
 import torch.utils.model_zoo as model_zoo
 __all__ = ['AlexNet', 'alexnet']
 
 
 model_urls = {
      'alexnet': 'https://download.pytorch.org/models/alexnet-owt-4df8aa71.pth',
  }
 
 class AlexNet(nn.Module):
      def __init__(self, num_classes=1000):
          super(AlexNet, self).__init__()
          self.features = nn.Sequential(
              nn.Conv2d(3, 64, kernel_size=11, stride=4, padding=2),
              nn.ReLU(inplace=True),
              nn.MaxPool2d(kernel_size=3, stride=2),
              nn.Conv2d(64, 192, kernel_size=5, padding=2),
              nn.ReLU(inplace=True),
              nn.MaxPool2d(kernel_size=3, stride=2),
              nn.Conv2d(192, 384, kernel_size=3, padding=1),
              nn.ReLU(inplace=True),
              nn.Conv2d(384, 256, kernel_size=3, padding=1),
              nn.ReLU(inplace=True),
              nn.Conv2d(256, 256, kernel_size=3, padding=1),
              nn.ReLU(inplace=True),
              nn.MaxPool2d(kernel_size=3, stride=2),
          )
          **self.classifier = nn.Sequential(
              nn.Dropout(),**
              nn.Linear(256 * 6 * 6, 4096),
              nn.ReLU(inplace=True),
              nn.Dropout(),
              nn.Linear(4096, 4096),
              nn.ReLU(inplace=True),
              nn.Linear(4096, num_classes),
          )
 
      def forward(self, x):
          x = self.features(x)
          **x = x.view(x.size(0), 256 * 6 * 6)**
          x = self.classifier(x)
          return x
 
  def alexnet(pretrained=False, **kwargs):
      r"""AlexNet model architecture from the
      `"One weird trick..." <https://arxiv.org/abs/1404.5997>`_ paper.
 
      Args:
          pretrained (bool): If True, returns a model pre-trained on ImageNet
      """
Can you pls update the wheel distribution accordingly.

**Comment by fmassa** [comment]
Yes, we will be cutting a new release of torchvision this week, and it's being tracked in https://github.com/pytorch/vision/issues/708 so I'm closing this as a duplicate
